packages/client/test/requestManager.spec.ts

```diff
@@ -1,15 +1,12 @@
 import { mnemonicToAccount } from 'viem/accounts';
+import { expect, describe, it, beforeEach } from '@windingtree/sdk-test-utils';
 import {
-  expect,
-  describe,
-  it,
-  beforeEach,
-  createRequest,
-  createOffer,
-} from '@windingtree/sdk-test-utils';
-import { generateMnemonic } from '@windingtree/sdk-utils/wallet';
-import { supplierId as spId } from '@windingtree/sdk-utils/uid';
-import { nowSec } from '@windingtree/sdk-utils/time';
+  createRandomRequest,
+  createRandomOffer,
+} from '@windingtree/sdk-messages';
+import { generateMnemonic } from '@windingtree/sdk-utils';
+import { supplierId as spId } from '@windingtree/sdk-utils';
+import { nowSec } from '@windingtree/sdk-utils';
```

Comment:
can be done, `import {generateMnemonic, supplierId as spId, nowSec} from '@windingtree/sdk-utils'`